Gems/GraphCanvas/Code/Source/Components/Slots/Execution/ExecutionSlotLayoutComponent.cpp

```diff
@@ -12,6 +12,8 @@
 
 #include <Components/Slots/Execution/ExecutionSlotConnectionPin.h>
 
+#include <Components/Slots/Execution/ExecutionSlotComponent.h>
```

Comment:
Not sure why these includes starting with `<Components...` have newlines between them, can you put them together? It's common practice to separate groups of includes (e.g. from Qt includes, AzCore, AzFramework, etc...) but these are in the `GraphCanvas` gem so can be grouped together

Reply:
And also in alphabetical order as well :arrows_counterclockwise: